licenses/licenses_tests.py

```diff
@@ -231,3 +231,49 @@ def test__licenses__views__Filled_licenseFile__4(
     browser.open(print_url(lr.id))
 
     assert "There is no profile" in browser.contents
+
+
+def test__licenses__admin__LicenseRequestAdmin__1(
+        browser, user, license_request, license_template_dict):
+    """Confirm multiple LRs."""
+    license_request.confirmed = 1
```

Comment:
```suggestion
    license_request.confirmed = True
```
As this is a boolean field, we should use it as one.